fabfile.py

```diff
@@ -19,6 +19,7 @@
     product_install,
     remove_katello_agent,
     run_errata,
+    satellite5_product_install,
```

Comment:
Good catch, but other missing tasks should be added in a separated commit or even another PR.